ui/raidboss/data/04-sb/raid/o3s.js

```diff
@@ -57,49 +86,54 @@ export default {
 
         data.holyTargets = data.holyTargets || [];
         data.holyTargets.push(matches.target);
-        return data.holyTargets.length === 4;
-      },
-      alarmText: (data, _matches, output) => {
-        if (data.holyTargets[1] !== data.me)
-          return '';
-        return output.stackOnYou();
+        return true;
       },
-      alertText: (data, _matches, output) => {
-        if (data.holyTargets[1] === data.me)
+      delaySeconds: 0.3,
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          othersStackOnHoly: {
+            en: '(others stack on ${player})',
+            de: 'andere stacken auf ${player}',
+            fr: 'Les autres se packent sur ${player}',
+            ja: '他は${player}と頭割り',
+            cn: '其他分摊${player}',
+            ko: '${player} 다른 쉐어징',
+          },
+          getOut: {
```

Comment:
Unrelated but this really needs to become an output, or we can use `Outputs.out` here, but I don't feel like it's the same, or this can be `Outputs.spread`?

Reply:
Spread sounds good, thanks!